src/cascade/input_data/configuration/construct_country.py

```diff
@@ -170,7 +171,7 @@ def covariate_to_measurements_nearest_favoring_same_year(measurements, sex, cova
             zip(
                 covariates[["age_lower", "age_upper"]].mean(axis=1) / 240,
                 covariates[["time_lower", "time_upper"]].mean(axis=1),
-                covariates["x_sex"],
+                covariates["sex_id"],
```

Comment:
This seems unlikely to work. Is this function even used? The problem is that this KDTree thing relies on having data points assigned to -0.5, 0, 0.5. If they are assigned to 1, 2, 3, they are distributed spatially in a different way. Really changes what this does.

Reply:
I'm walking this back. It might still work. The idea was that a data point at x_sex=0.0 would be close to 0.5 and -0.5, if they were missing. But a point at 3 is still closest to missing values at 1 and 2 if they are gone.

Reply:
It is not used. Should I just dump it?